doto-frontend/src/components/pages/Calendar/Calendar.js

```diff
@@ -62,7 +62,7 @@ const Calendar = () => {
         };
         fetchUserInfo();
         fetchTasks();
-    }, []);
+    });
```

Comment:
I don't think removing the `[]`  parameter is a good idea because the useEffect hook will be called on _every_ render since we are effectively passing in `undefined` as the second param.

 The empty array meant that it only rendered when the component mounts.

Reply:
Okay, but the `[]` causes the build to fail, so I don't think it can be kept, irrespective of rendering optimisations. Based on the documentation [here](https://reactjs.org/docs/hooks-effect.html), I also don't see *how* an empty array actually serves to produce the optimisation you're talking about.
In any case, the original warning suggested either:
* Removing the empty array
* Adding `setTheme`

So I'll implement the later.